### ISSUE TITLE
Getting error when using local_auth with flutter

### ISSUE THREAD

**Issue by appcentuate** [tekartik/sqflite#895]
handler.post(this::runQueuedOperations);
                                   ^
  (use -source 8 or higher to enable method references)
C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\sqflite-2.2.0+1\android\src\main\java\com\tekartik\sqflite\Database.java:243: error: lambda expressions are not supported in -source 1.7
        wrapSqlOperationHandler(operation, () -> doQuery(operation));
                                              ^
  (use -source 8 or higher to enable lambda expressions)
C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\sqflite-2.2.0+1\android\src\main\java\com\tekartik\sqflite\SqflitePlugin.java:197: error: lambda expressions are not supported in -source 1.7
        handler.post(() -> {
                        ^
  (use -source 8 or higher to enable lambda expressions)

**Comment by alextekartik** [comment]
Yes sorry. This was reported in #891 You need to enable Java 8 in your gradle. Performing a `flutter pub upgrade` to update sqflite to 2.2.0+3 or later should also fix your issue.
